vmdk_plugin/plugin.go

```diff
@@ -340,9 +346,6 @@ func (d *vmdkDriver) Path(r volume.Request) volume.Response {
 func (d *vmdkDriver) Mount(r volume.MountRequest) volume.Response {
 	log.WithFields(log.Fields{"name": r.Name}).Info("Mounting volume ")
 
-	d.m.Lock()
-	defer d.m.Unlock()
```

Comment:
maybe I am missing something, but won't it be now possible to get refcnt (libe 354)m then another thread to change the actual refcount before decision on line 356 is made ? Or is there something else (e.g. Docker serializing of mounts) than would prevent it ? If there is, **please add a comment to the code explaining it** .